engine/classes/Elgg/HooksRegistrationService.php

```diff
@@ -243,4 +248,36 @@ protected function getMatcher($spec) {
 
 		return new MethodMatcher($spec[0], $spec[1]);
 	}
+
+	/**
+	 * Temporarily remove all event/hook registrations (before tests)
+	 *
+	 * Call backup() before your tests and restore() after.
+	 *
+	 * @note This behaves like a stack. You must call restore() for each backup() call.
+	 *
+	 * @return void
+	 * @see restore
+	 * @access private
+	 * @internal
+	 */
+	public function backup() {
```

Comment:
I think so. This is internal API and I expect our devs will be willing to check the docs to see what it does.